### PR TITLE
docs: update README.md

### LINKED ISSUE #25: Add docs for the ics contracts
Lack of documentation for https://github.com/synapseweb3/ibc-ckb-contracts/tree/8e04df93ab30c6ec94c8aa520105d6a8ffb6d5bd/contracts,
need some descriptions for the ics contracts, at least, explain how to use them.

### PR DESCRIPTION
**Pull request by ashuralyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 > :warning: **WARNING** This repository is still in the proof-of-concept stage.
 
-[IBC] implementations in [CKB] contracts.
+This project consists of two CKB contracts serving for `Forcerelay/Eth` and `Forcerelay/Axon` respectively.
```
Comment (Flouse) [inline README.md]:
```suggestion
This project consists of two types of CKB contracts serving for `Forcerelay/Eth` and `Forcerelay/Axon` respectively.
```

**Review by Flouse — CHANGES_REQUESTED** [review]

**Comment by Flouse** [comment]
According to the recent discussion, I renamed this repo to `synapseweb3/crosschain-ckb-contracts`.

**Comment by ashuralyk** [comment]
> According to the recent discussion, I renamed this repo to `synapseweb3/crosschain-ckb-contracts`.

What about `forcerelay-ckb-contracts`?

**Comment by Flouse** [comment]
> What about `forcerelay-ckb-contracts`?

changed

**Review by Flouse — APPROVED** [review]

**Comment by Flouse** [comment]
> What about `forcerelay-ckb-contracts`?

Please change this readme's h1 title too?
https://github.com/synapseweb3/forcerelay-ckb-contracts/pull/28/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1

**Review by linnnsss — APPROVED** [review]

**Pull request merged**